build-system/tasks/helpers.js

```diff
@@ -295,7 +285,8 @@ async function compileMinifiedJs(srcDir, srcFilename, destDir, options) {
     }
     endBuildStep('Minified', name, timeInfo.startTime);
 
-    if (!argv.noconfig && MINIFIED_TARGETS.includes(minifiedName)) {
+    const target = path.basename(minifiedName, path.extname(minifiedName));
```

Comment:
You can remove the path.extname part, that's how path.basename works to begin with

Reply:
Nope, it's required according to the [docs](https://nodejs.org/api/path.html#path_path_basename_path_ext).

Also confirmed locally:

![image](https://user-images.githubusercontent.com/26553114/81860503-d7d32b00-9534-11ea-8a17-ede6e2afcd82.png)

Reply:
Oops, I tested it and somehow didn't realize you're correct :D